x-pack/test/alerting_api_integration/spaces_only/tests/alerting/builtin_alert_types/index_threshold/alert.ts

```diff
@@ -135,7 +135,8 @@ export default function alertTests({ getService }: FtrProviderContext) {
       }
 
       // there should be 2 docs in group-0, rando split between others
-      expect(inGroup0).to.be(2);
```

Comment:
In these tests, there should be two actions scheduled for every alert interval, one in one "group", the other in a different one, and we wait for 2 turns of the alert interval.  There are always three groups of data in the index, but `termSize` will limit only two groups being returned - the "top" one, and one of the others (which we don't care about, as it's random-ish).  For count (this test), group-0 has more docs, and so should be present in the output.  For the sum (below), group-3 has larger values, and so should be present.  For min (below sum), group-0 has smaller values, and so should be present.

While there _should_ be 2 docs in the "top" group, the test failure this was written for got 3 at one point.  Seems to be a very-rarely-flaky situation, so the check was changed from `=== 2` to `> 0`, to account for slop in the opposite direction as well (only getting 1 instead of 2).